src/hip/hip_build_utils.cpp

```diff
@@ -79,6 +82,39 @@ char* generateFlagValue(MIOPEN_CMP_Flags_t x)
     return (char*)&(MIOPEN_CMP_FLAGS[x]);
 }
 
+// API to get MIOPEN AMD GCN Assembler Path Values.
+const char* getAMDGCNAssemblerPath()
```

Comment:
Please put this and three subsequent functions into `miopen` namespace.

Reply:
If you put them in the miopen namespace they are no longer available to call from C.

Reply:
You may not use them in C, but the question is "what is the interface you are wanting to expose?"